README.md

```diff
@@ -9,6 +9,9 @@ By installing the MicroProfile Extension Pack, the following extensions are inst
 * [MicroProfile Starter](https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.mp-starter-vscode-ext) - Quickly generate a MicroProfile Maven project.
 * [MicroProfile Rest Client Generator](https://marketplace.visualstudio.com/items?itemName=MicroProfile-Community.mp-rest-client-generator-vscode-ext) - Quickly generate a MicroProfile Rest Client interface template from an OpenAPI document.
```

Comment:
Should `MicroProfile Rest Client Generator` should be moved to `Developing a MicroProfile project`? Since it is not really creating an entire project, but is just for creating the rest client in an existing project? 

Im also fine with leaving it under `Creating a MicroProfile project`.

Reply:
I associate the REST Client Generator more with project generation, so I would prefer to leave it under `Creating a MicroProfile project`.

@yeekangc any thoughts?

Reply:
I am fine to leave it where it is for now.  Thanks.